cumulus/pallets/parachain-system/src/lib.rs

```diff
@@ -1418,6 +1429,21 @@ impl<T: Config> Pallet<T> {
 		CustomValidationHeadData::<T>::put(head_data);
 	}
 
+	/// Send the ump signals
+	#[cfg(feature = "experimental-ump-signals")]
+	fn send_ump_signals() {
```

Comment:
Just one signal for now.

```suggestion
	fn send_ump_signal() {
```